web-ui/src/main/resources/catalog/templates/admin/harvest/type/ogcwxs.html

```diff
@@ -53,7 +53,7 @@
               data-ng-model="harvesterSelected.options.serviceTemplateUuid"
               data-ng-options="t['geonet:info'].uuid as t.getTitle() for t in ogcwxsTemplates">
       </select>
-      <p class="help-block" data-translate="">ogcwxs-serviceTemplateUuidHelp</p>
+      <p class="help-block" data-translate="" translate-values="{metadataTemplateType: metadataTemplateType}">ogcwxs-serviceTemplateUuidHelp</p>
```

Comment:
`serviceTemplateUuidHelp` does not have metadataTemplateType placeholder. So you can remove it ?